### ISSUE TITLE
AI/Cyborg Security Hud doesn't show loyalty implants

### ISSUE THREAD

**Issue by TheWiznard** [tgstation/-tg-station#16754]
Server revision compiled on: 2016-04-08
Test merge active of PR #16718 
Based off master commit e3a337efd6f6a2ed384e5efa2a0e487231258377 

![no implants](https://cloud.githubusercontent.com/assets/9158683/14401395/5bb9aeda-fdd6-11e5-9a2d-55eb514b6a0b.gif)

What happens:
The AI/borg sec hud only shows the guys job icon of their ID. so like this guy the jaintor would show up as the jaintor icon if he was wearing an ID. Everyone who doesn't have an ID only shows this question mark icon.  You also can't tell if someone is implanted or not.

What you think should happen:
The AI/borg sec hud should show if someone is implanted or not. This image might not be the best of what I'm trying to show here.

tl;dr there is no way of telling if someone is implanted or not while using the ai/borg sechud.

**Comment by KazeEspada** [comment]
Because its not a sec implant, its just a job UI

**Comment by TheWiznard** [comment]
no, like. even if they are 100% implanted you can't see them. I know the guy in the gif isn't implanted, so I guess it actually contributes nothing here. During the round this was taken I watched the captain and security literally implant every person on the station (50+) people and I still couldn't see single implanted person

**Comment by PKPenguin321** [comment]
yes because it is a job HuD not a sec HuD
this is intentional so that AIs have less security bias

**Comment by TheWiznard** [comment]
Well then rename it to job hud and not security HuD then
